examples/FirebaseStream_ESP8266/FirebaseStream_ESP8266.ino

```diff
@@ -42,22 +42,22 @@ void setup() {
   Serial.println();
   Serial.print("connected: ");
   Serial.println(WiFi.localIP());
-  
-  fbase.stream("/bitcoin");
 }
 
 
-void loop() {
-  if (fbase.error()) {
+void loop() {  
+  static FirebaseEventStream stream = fbase.stream("/bitcoin");
```

Comment:
I didn't realize this would force the user to store a `static`, I think we should find a way that keep the state in the Firebase object.